scripts/simulation/chat.ts

```diff
@@ -0,0 +1,67 @@
+import { strict as assert } from "assert";
+import chalk from "chalk";
+
+import { SimulatorContext } from "../simulator";
+
+import { sendBotVenueMessage as actualSendBotVenueMessage } from "../lib/bot";
+import { withErrorReporter } from "../lib/log";
+import { sleep } from "../lib/utils";
+
+export const DEFAULT_CHAT_CHUNK_SIZE = 100;
+export const DEFAULT_CHAT_TICK_MS = 1000;
+export const DEFAULT_CHAT_AFFINITY = 0.005;
```

Comment:
I have taken care to compartmentalize simulation specific constants into their own files as to not create unnecessary references.